apitools/base/py/base_api.py

```diff
@@ -376,6 +378,15 @@ def num_retries(self, value):
         self.__num_retries = value
 
     @property
+    def check_response_func(self):
```

Comment:
I figured it would be easier to use if you can set it set after instantiating a generated api. Otherwise generated api has to plumb it through in its constructor which may be annoying.